package.json

```diff
@@ -6,9 +6,21 @@
     "url": "https://github.com/nsoss/srce-desktop-app.git"
   },
   "scripts": {
-    "start": "electron ."
+    "start": "electron .",
+    "build": "webpack"
   },
   "devDependencies": {
-    "electron": "4.1.4"
+    "autoprefixer": "^9.5.1",
+    "bootstrap": "^4.3.1",
```

Comment:
Lots of these stuff needs to be in `dependencies`, not in `devDependencies`, also is `jquery` necessary?

Reply:
devDependencies because of it used only for development. Webpack will generate App.js where we will have all JS code in one file.

Reply:
Im pretty sure that `bootstrap` will be used in production, amongst some other `npm` packages included.

Reply:
[Here](https://github.com/facebook/create-react-app/issues/6180#issuecomment-453640473) is a great comment on this topic by Dan Abramov.